gradle.properties

```diff
@@ -1,3 +1,4 @@
 android.useAndroidX=true
 android.enableJetifier=true
-android.enableR8=false
\ No newline at end of file
+android.enableR8=false
```

Comment:
Does this disable minification?

Reply:
No it just means that proguard will be used, rather than R8